crates/uv-cli/src/lib.rs

```diff
@@ -1586,6 +1595,14 @@ pub struct PipInstallArgs {
     #[arg(long, overrides_with("no_deps"), hide = true)]
     pub deps: bool,
 
+    /// Install the specified dependency group in the specified pyproject.toml
+    ///
+    /// If no path is provided, ./pyproject.toml is used.
```

Comment:
Does this respect `--project`? It seems important that it does.

Reply:
We should also probably call out that `-r .../pyproject.toml` and `<path>` do not affect this?

Reply:
I feel strongly that the default path should be relative to `--project`.

Reply:
(and `--directory`)